src/plugin/vault/etc/setup-vault.sh

```diff
@@ -57,8 +57,11 @@ rm /tmp/vault-plugins/$PLUGIN_NAME || true
 set -o pipefail
 
 # Build the plugin binary
-SCRIPT_DIR=$(dirname "${0}")
-go build -o /tmp/vault-plugins/$PLUGIN_NAME ${SCRIPT_DIR}/..
+(
+  cd ${GOPATH}/src/github.com/pachyderm/pachyderm/src/plugin/vault && \
+  make plugin
+)
```

Comment:
Why does this need to run in a subshell?

Reply:
Just to avoid having to `cd` back to the current directory